modules/rules_jsonnet/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "homepage": "https://github.com/bazelbuild/rules_jsonnet",
+  "maintainers": [
+    {
+      "email": "bcr-maintainers@bazel.build",
```

Comment:
Do you want to add yourself as a module maintainer?

Reply:
Sure! In fact, I would also be interested in becoming a maintainer of the actual project. There are various changes/improvements I would like to make to that rule. The fact that it currently takes a long time for PRs to be processed is making it unappealing to send those in.

Reply:
Cc @sparkprime @meteorcloudy

Reply:
@EdSchouten I think it should be fine add you as a maintainer, thanks for the interest! But let me confirm with @sparkprime first. And probably we should move it to bazel-contrib in future, I'm working on that for repos under bazelbuild that are in fact community maintained.

Reply:
@EdSchouten Now you have write access to https://github.com/bazelbuild/rules_jsonnet